### DISCUSSION TITLE
元数据丢失或者不一致的问题

### DISCUSSION THREAD

**Discussion by stgztsw** [apache/doris#17096] [A - General / Q&A]
doris版本是1.0.0。配置是3个FE节点(一个mater，2个follower；FE前面加了nginx负载均衡)和3个be节点。最近发现了一个比较奇怪的问题。我们对某张表新增了字段。当天都正常，可以写数据，可以查数据。并且搭建了报表也能正常运行。但是到了第二天报表报错了，发现新增的没有了(我们确认没有人删除过字段)。并且类似的事情发生不止一次了。这边的想法是是不是由于3个fe节点之间元数据不一致

**Comment by boluor** [comment]
两个点，一个是可以联系我们看看日志来分析的，一个是建议升级到1.1系列中的稳定版本1.1.5，从你当前的版本可以很方便的升级上去观察下。
